File: PresentationLayer/PresentationLayer/Models/Campus.cs
namespace PresentationLayer.Models
{
    public class Campus
    {
    
        public string City { get; set; }
        public string Street { get; set; }
        public string PostCode { get; set; }
        public string Name { get; set; }

        public Campus(string city, string street, string postCode, string name)
        {
            City = city;
            Street = street;
            PostCode = postCode;
            Name = name;
        }
    }
}

File: ApplicationLogic/src/main/java/dk/via/sep3/group1/applicationlogic/model/Campus.java
package dk.via.sep3.group1.applicationlogic.model;

import com.fasterxml.jackson.annotation.JsonProperty;

public class Campus {
    @JsonProperty("city")
    private String city;
    @JsonProperty("street")
    private String street;
    @JsonProperty("postCode")
    private String postCode;
    @JsonProperty("name")
    private String name;

    public Campus() {
    }

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }

    public String getStreet() {
        return street;
    }

    public void setStreet(String street) {
        this.street = street;
    }

    public String getPostCode() {
        return postCode;
    }

    public void setPostCode(String postCode) {
        this.postCode = postCode;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}
